charts/openebs/templates/deployment-admission-server.yaml

```diff
@@ -47,6 +47,10 @@ spec:
       containers:
         - name: admission-webhook
           image: "{{ .Values.image.repository }}{{ .Values.webhook.image }}:{{ .Values.webhook.imageTag }}"
+{{- if .Values.webhook.resources }}
+          resources:
+{{ toYaml .Values.webhook.resources | trimSuffix "\n" | indent 12 }}
```

Comment:
@Mosibi any specific reason the indent is set as 12 not 10 ?

Reply:
The indentation needs 2 extra spaces in that specific file, see the the line with indent 12 and the line above it in the output below 
````
$ grep -B1 trimSuffix templates/*|sed -e 's/template.*yaml[:-]//g'
        resources:
{{ toYaml .Values.ndm.resources | trimSuffix "\n" | indent 10 }}
--
          resources:
{{ toYaml .Values.webhook.resources | trimSuffix "\n" | indent 12 }}
--
        resources:
{{ toYaml .Values.localprovisioner.resources | trimSuffix "\n" | indent 10 }}
--
        resources:
{{ toYaml .Values.apiserver.resources | trimSuffix "\n" | indent 10 }}
--
        resources:
{{ toYaml .Values.provisioner.resources | trimSuffix "\n" | indent 10 }}
--
        resources:
{{ toYaml .Values.snapshotOperator.controller.resources | trimSuffix "\n" | indent 10 }}
--
        resources:
{{ toYaml .Values.snapshotOperator.provisioner.resources | trimSuffix "\n" | indent 10 }}
--
        resources:
{{ toYaml .Values.ndmOperator.resources | trimSuffix "\n" | indent 10 }}
````

Reply:
got it thanks